x/claims/keeper/claim.go

```diff
@@ -189,14 +189,19 @@ func (k Keeper) MergeClaimsRecords(
 			amtIBCRecipient := k.GetClaimableAmountForAction(ctx, recipientClaimsRecord, action, params)
 			amtIBCSender := k.GetClaimableAmountForAction(ctx, senderClaimsRecord, action, params)
 			claimableAmt = claimableAmt.Add(amtIBCRecipient).Add(amtIBCSender)
-			mergedRecord.ActionsCompleted[i-1] = true
+			mergedRecord.ClaimAction(action)
 		}
 	}
 
+	// safety check to prevent error while sending coins from the module escrow balance to the recipient
+	if claimableAmt.IsZero() {
+		return mergedRecord, nil
+	}
+
 	claimedCoins := sdk.Coins{{Denom: params.ClaimsDenom, Amount: claimableAmt}}
```

Comment:
Would rename `claimableAmt` to `claimedAmt`. Claimable sound to me like the coins are unclaimed